usr/lib/sticky/common.py

```diff
@@ -233,20 +241,33 @@ def restore_backup(self, menuitem, window):
 
         dialog.destroy()
 
+    def check_json(self,file):
+        for group in file:
+            for note in file[group]:
+                if 'x' not in note or 'y' not in note:
+                    raise ValueError("No coordinate value found (x,y)")
+                if 'height' not in note or 'width' not in note:
+                    raise ValueError("No height or width value found")
```

Comment:
We really only need title, text, and color to be there. The position and size info is not used in the manager, and is auto-generated by the note if it doesn't already exist. However, we might want to make sure the structure of the file is correct (ie the root item should be a dict, with group names as keys, and the values being lists of dicts).